powerauth-fido2-tests/src/main/java/com/wultra/security/powerauth/fido2/service/AssertionService.java

```diff
@@ -81,20 +77,36 @@ public AssertionOptionsResponse assertionOptions(final AssertionOptionsRequest r
         final List<CredentialDescriptor> existingCredentials = credentialList
                 .orElse(Collections.emptyList())
                 .stream()
-                .map(AssertionService::toCredentialDescriptor).toList();
+                .map(AssertionService::toCredentialDescriptor)
+                .collect(Collectors.toCollection(ArrayList::new));
```

Comment:
What is the reason not to use just `.toList()`? To make it mutable and add an item later on?

Reply:
yes. In case a credential with 'usb' transports is detected in the list later, a new credential holding the operation data is appended